plugins/modules/ec2_asg_instance_refresh.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: ec2_asg_instance_refresh
+version_added: 3.2.0
+short_description: Start or cancel an EC2 Auto Scaling Group (ASG) instance refresh in AWS
+description:
+  - Start or cancel an EC2 Auto Scaling Group instance refresh in AWS.
+  - Can be used with M(community.aws.ec2_asg_instance_refresh_info) to track the subsequent progress.
+author: "Dan Khersonsky (@danquixote)"
+options:
+  state:
+    description:
+      - Desired state of the ASG.
+    type: str
+    required: true
+    choices: [ 'started', 'cancelled' ]
+  name:
+    description:
+      - The name of the auto scaling group you are searching for.
+    type: str
+    required: true
+  strategy:
+    description:
+      - The strategy to use for the instance refresh. The only valid value is C(Rolling).
```

Comment:
if that is the only valid value, does it make sense to add `choices: [ 'Rolling' ]` to make it clear? im not sure ive never seen a param that only has one valid value

Reply:
Based on the [API doc](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/autoscaling.html#AutoScaling.Client.start_instance_refresh), currently `Rolling` is the only possible value.
IMO currently we could not add `choices` as there's only single option, but potentially change that, if/when the API provides more choices for `Strategy`.